tests/test_dataframe_reader_extensions.py

```diff
@@ -13,11 +29,6 @@ def test_load():
     StructField("infection_case",StringType()), \
     StructField("confirmed",     IntegerType()), \
     StructField("latitude",      FloatType()), \
-    StructField("cilongitudety", FloatType()) \
+    StructField("longitude", FloatType()) \
```

Comment:
align FloatType() with previous line by adding 4 spaces, just to keep readability